docs/geth-pos-privnet.md

```diff
@@ -10,45 +10,51 @@ Special thanks to Mateusz ([@mmrosum](https://twitter.com/mmrosum)) who put the
     1. `ethkey generate` + geth’s —nodekey
     2. geth console + eth.newAccount
+    3. create a new address in something like metamask, and copy the private key.  inside a geth console session (`geth --datadir ~/.ethereum/local-testnet/testnet/geth-node-1 console`), run `web3.personal.importRawKey("<Private Key>","<Password>")`
 7. Initialize geth from the json: `geth init --datadir ~/.ethereum/local-testnet/testnet/geth-node-1 ~/path/to/ethereum-genesis-generator/data/el/geth.json`
 8. Check the node starts to mine and kill it quickly
 You only have 100 blocks until fork is enabled and 400 blocks until node stops mining
-`geth --datadir ~/.ethereum/local-testnet/testnet/geth-node-1 --networkid 4242 --http --http.port 8545 --discovery.dns "" --port 30303 --mine --miner.etherbase=<address> --miner.threads 1 --miner.gaslimit 1000000000 --miner.maxmergedbundles 1 --unlock "<address>" --password <(echo "<password>") --allow-insecure-unlock > ~/.ethereum/miner.log 2>&1`
+`geth --datadir ~/.ethereum/local-testnet/testnet/geth-node-1 --networkid 4242 --http --http.port 8545 --http.api personal,eth,net,web3,engine,debug --discovery.dns "" --port 30303 --mine --miner.etherbase=<address> --miner.threads 1 --miner.gaslimit 1000000000 --unlock "<address>" --password <(echo "<password>") --allow-insecure-unlock > ~/.ethereum/miner.log 2>&1`
+where `<address>` is the public key of the wallet you created in step (6)
 
 ### Prepare PoS chain
 
 1. Clone https://github.com/sigp/lighthouse
 2. Go to `scripts/local_testnet`
     1. Modify vars.env:
-        1. Set `ETH1_NETWORK_MNEMONIC, DEPOSIT_CONTRACT_ADDRESS, GENESIS_FORK_VERSION` to be the same as in PoW’s config
+        1. Set `ETH1_NETWORK_MNEMONIC, DEPOSIT_CONTRACT_ADDRESS, GENESIS_FORK_VERSION` to be the same as in PoW’s config (GENESIS_FORK_VERSION is in `ethereum-genesis-generator/data/cl/config.yaml`)
         2. Set `GENESIS_DELAY` to 30
         3. Set `ALTAIR_FORK_EPOCH` to 1
         4. Add `MERGE_FORK_EPOCH=1`
         5. Adjust `SECONDS_PER_SLOT, SECONDS_PER_ETH1_BLOCK, BN_COUNT`to your preference
             1. There seem to be some issues with multiple beacon nodes using the same EL, if doesn’t work set `BN_COUNT` to 1
         6. Do not change `VALIDATOR_COUNT, GENESIS_VALIDATOR_COUNT` to less than 64
+        7. modify VC_ARGS line to `VC_ARGS="--suggested-fee-recipient <address>"`, where <address> is the same public key that you registered in the `geth` command #8 above
     2. Modify scripts/local_testnet/beacon_node.sh:
-        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints [http://127.0.0.1:8545](http://127.0.0.1:8545/) --execution-endpoints [http://127.0.0.1:8545](http://127.0.0.1:8545/) --http-allow-sync-stalled`
+        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints http://127.0.0.1:8545/ --execution-endpoints http://127.0.0.1:8545/ --http-allow-sync-stalled`
```

Comment:
```suggestion
        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints http://127.0.0.1:8545/ --execution-endpoints http://127.0.0.1:8551/ --http-allow-sync-stalled`
```

I think you need to use port 8551 for Geth's engine API. The :8545 endpoint is unencrypted and AFAIK won't expose the engine API (certainly not by default).

Reply:
You'll need to establish a JWT for the connection in this case. If you want to follow the path of least resistance I'd aim to minimise differences between Geth and Lighthouse's config here, and the config used on public merge testnets, e.g. Ropsten: https://github.com/remyroy/ethstaker/blob/main/merge-ropsten.md#configuring-your-geth-node